lib/components/table.vue

```diff
@@ -76,14 +76,15 @@
 </template>
 
 <script>
-    import {warn} from '../utils';
+    import { warn } from '../utils';
+    import { keys } from '../utils/object';
```

Comment:
@tmorehouse, the destructuring is just the named export access syntax. Looks like a duck, walks like a duck, but it's a module.